verrazzano/verrazzano
# VZ-8892 Update the Thanos VMC sync to remove Prometheus federation

**Changes**
- Add a cluster label to the managed cluster store targets for selection
- Change the update and create process to factor in labels
- Add an ingress host flag to override the Rancher API calls for running in IDE
- Update and add unit tests to test new update and delete process

**Testing**
- Run a multicluster setup in kind
- Turn on and off the Thanos component on the managed cluster
- Turn on and off the Thanos component on the admin cluster
- Make sure that the Thanos query store configmap gets modified correctly
- Make sure that the Prometheus additional configs get modified correctly
- Deregister and make sure both Thanos and Prometheus targets get cleaned up
